dataprofiler/labelers/base_data_labeler.py

```diff
@@ -58,6 +58,8 @@ def __eq__(self, other):
         :return: Whether or not self and other are equal
         :rtype: Bool
         """
+        if other is None:
+            return False
```

Comment:
None catching in `BaseDataLabeler.__eq__`

Reply:
should this instead be an `isinstance` check?

Reply:
Changed it